Release Notes/Release Notes v2.11.0-beta1.md

```diff
@@ -0,0 +1,42 @@
+# .NET Driver Version 2.11.0-beta1 Release Notes
+
+This is a beta release for the 2.11.0 version of the driver.
+
+The main new features in 2.11.0-beta1 are:
+
+* .NET Standard 2.0 support
```

Comment:
FWIW the loose rule of thumb I've been following is that when the bullet is a full sentence, I include a period, else I don't

Reply:
My point was that we probably don't always follow for any rule here (looking at the link in my previous message). Maybe we need to mention it (whatever it will be) somewhere?

Reply:
Let's go with this: https://style.mla.org/vertical-lists-in-mla-style/
Basically the rule is: all fragments, or all sentences. In this case, all the bullets are fratments
Since our lead in sentence is "The main new features in 2.11.0-beta1 are:" 
it makes sense to me that we'd have a list of noun clauses (fragments).